staging/src/k8s.io/apiserver/pkg/endpoints/handlers/update.go

```diff
@@ -89,8 +89,14 @@ func UpdateResource(r rest.Updater, scope RequestScope, admit admission.Interfac
 		}
 		defaultGVK := scope.Kind
 		original := r.New()
+
+		hubGroupVersion := scope.HubGroupVersion
+		if hubGroupVersion.Empty() {
+			hubGroupVersion = schema.GroupVersion{Group: defaultGVK.Group, Version: runtime.APIVersionInternal}
```

Comment:
same here. Let's enforce non-emptiness in the scope. Then all the handler changes become much simpler.

Reply:
Done